fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/IcebergMetadataOps.java

```diff
@@ -173,4 +177,7 @@ public void dropTable(DropTableStmt stmt) throws DdlException {
         catalog.dropTable(TableIdentifier.of(dbName, tableName));
         db.setUnInitialized(true);
     }
+
```

Comment:
Unnecessary modifications

Reply:
During the modification process, I accidentally pressed a blank line